pkg/engine/cmd.go

```diff
@@ -17,7 +17,8 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/google/shlex"
+	// "github.com/anmitsu/go-shlex"
```

Comment:
nit: remove this and other comments

Reply:
ah, thought I dropped these. Thanks for catching. done.